pyproject.toml

```diff
@@ -94,7 +94,10 @@ pcodec = [
     "pcodec>=0.3,<0.4",
 ]
 crc32c = [
-    "google-crc32c>=1.5",
+    "crc32c>=2.7",
+]
+google_crc32c = [
+    "google-crc32c>=1.5"
```

Comment:
neither of these are installed by default, right?

Reply:
fixed in [`214a073` (#802)](https://github.com/zarr-developers/numcodecs/pull/802/commits/214a07359871cca23248132c567a19474e728168)